meetings/2024-10-02.md

```diff
@@ -0,0 +1,109 @@
+* RG: I think the larger conversation is what kind of a social infrastructure we want to build to bring all these people together. There is some merit to UC thing but we should be more open to out of box thing. What we think of the best possible system because we might not know what - just not necessarily the way LWS is thinking UCs. But be more broader / experimental.
+* MdJ: I like that. I think we can also focus on interop between applications. That uses Solid and make those work together. They might not be UCs we imagine. That's what I'd consider outside-in. Starting from real examples.
+* eP: Coming back to LWS UCs re overlap. As I understand hte harter, the client-client is out of scope. There is not going to be domain-specific. Naturally the WG will not fully satisfy all the UCs because there is always the client-client part. That's why I think the UCs should be shared. Ensuring what we have fits into the LWS.
+* RG: There can be generic use cases, I'll point you to Douglas Engelbart in that respect. Use cases don't necessarily have to be domain-specific. We have a generic UC on what knowledge-workshop would be and thats a good light of how software should be designed.
+* MdJ: To me the topic of how different specs are interoperable is interesting. As an app dev, one might use one and dev another. If all those things are incubating all being interop under Solid.
+
+
+### Use cases
+
+URL: https://solid.github.io/authorization-panel/authorization-ucr/
+URL: https://github.com/solid/user-stories
+URL: https://github.com/solid/webid-profile/blob/main/notes/use-cases.md
+URL: https://solid.github.io/notifications-panel/notifications-ucr
+
+* eP: Authorization use case were written in reverse, starting from WAC as a solution plus some know limitations.
+* eP: Instead of making up use cases, we can work with existing communities, preferably open source and document their real world usecases which already have user base. As an example we had a meeting with https://openfoodnetwork.org/
+* eP: We can also try to "eat our own cookings" more. This prototype tries to define some CG meetings based requirements https://github.com/janeirodigital/sai-js/tree/main/examples/plenary#requirements
```

Comment:
```suggestion
* eP: Authorization use case were written in reverse, starting from WAC as a solution, plus some known limitations.
* eP: Instead of making up use cases, we can work with existing communities, preferably open source, and document their real world use cases, which already have a user base. As an example, we had a meeting with <https://openfoodnetwork.org/>
* eP: We can also try to "eat our own cooking" more. This prototype tries to define some CG meetings based requirements: <https://github.com/janeirodigital/sai-js/tree/main/examples/plenary#requirements>.
```